src/main/java/edu/kis/powp/jobs2d/shapes/lines/UserModifiableLine.java

```diff
@@ -0,0 +1,62 @@
+package edu.kis.powp.jobs2d.shapes.lines;
+
+import edu.kis.legacy.drawer.shape.line.AbstractLine;
+
+import java.awt.*;
+
+public class UserModifiableLine extends AbstractLine {
+
+    private static final float DEFAULT_THICKNESS = 1.0f;
+    private static final Color DEFAULT_COLOR = Color.BLACK;
+    private static final boolean DEFAULT_STYLE = false;
+
+    private UserModifiableLine(LineBuilder builder) {
+        this.color = builder.color;
+        this.dotted = builder.dotted;
+        this.thickness = builder.thickness;
+    }
+
+    public UserModifiableLine() {
+        this.thickness = DEFAULT_THICKNESS;
+        this.color = DEFAULT_COLOR;
+        this.dotted = DEFAULT_STYLE;
+    }
+
+    public static LineBuilder builder() {
+        return new LineBuilder();
+    }
+
+    public LineBuilder rebuilder() {
+        return new LineBuilder();
```

Comment:
It does not differ from a static builder() method. If it could pass current line params it would make sense (similar to instance cloning).

Reply:
Fixed in latest commit to master branch.